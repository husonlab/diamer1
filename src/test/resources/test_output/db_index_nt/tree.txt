parent id	node id	rank	label	kmers in database
	0	no rank	root	0
0	1	1 from root	rank1	113
0	2	1 from root	rank2	237
1	4	2 from root	organism2	336
1	5	2 from root	organism3	318
2	3	2 from root	organism1	82
