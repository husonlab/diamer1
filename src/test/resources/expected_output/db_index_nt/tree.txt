parent id	node id	rank	label	kmers in database
	0	no rank	root	0
0	1	1 from root	rank1	113
0	2	1 from root	rank2	236
1	4	2 from root	organism2	334
1	5	2 from root	organism3	317
2	3	2 from root	organism1	81
